deployment/environments/dev/values.yaml

```diff
@@ -1,6 +1,18 @@
-contracts: 0xD0b57307a4B9e0147F389ab99547E1530D323be0
-rpc: https://polygon-mumbai-bor.publicnode.com
-debug: true
-startingBlock: "41924130"
-blocksMargin: 70
-blocksRange: 2
+networks:
+  - name: polygon-mumbai
+    contracts: 0xD0b57307a4B9e0147F389ab99547E1530D323be0
+    rpc: https://polygon-mumbai-bor.publicnode.com
+    host: polygon-mumbai.unode.gorengine.com
+    debug: true
+    startingBlock: "41924130"
+    blocksMargin: 70
+    blocksRange: 2
+  - name: polygon-mainnet
+    contracts: 0xBC4CA0EdA7647A8aB7C2061c2E118A18a936f13D
+    rpc: https://eth.llamarpc.com
```

Comment:
same for secrets. also, is this really polygon mainnet rpc?

Reply:
same

Reply:
This endpoint must be changed for `https://polygon-bor.publicnode.com`

Reply:
We must change the values of both `rpc`.
Running the unode with that L1 mumbai node, the following error occurs:
`error filtering events: Please specify an address in your request or, to remove restrictions, order a dedicated full node here: https://www.allnodes.com/matic/host`

and the mainnet value refers to an L1 node of Ehtereum mainnet.

I suggest we use https://rpc-mumbai.maticvigil.com for mumbai and https://polygon.llamarpc.com for mainnet.
If you wish to check other public nodes, you can check this page: https://chainlist.org/?testnets=true&search=polygon